Jint/Runtime/Interop/SpreadParametersAttribute.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Jint.Runtime.Interop
+{
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class SpreadParametersAttribute : Attribute
```

Comment:
we don't use attributes for any other interop scenario at the moment in Jint so probably shouldn't use for this either. if changing logic would be breaking I think it's find as long as the behavior changes for the better (expected).

Reply:
I agree!